diesel_derives/Cargo.toml

```diff
@@ -16,7 +16,7 @@ proc-macro2 = "1"
 
 [dev-dependencies]
 cfg-if = "0.1.0"
-diesel = "1.4.0"
+diesel = { path = "../diesel" }
```

Comment:
This will cause problems with `cargo publish` as this is a local only dependency. (It does not help that this is a dev-dependency…) You need to also add an explicit version here (and for the other dev-dependencies too)